RELEASE-NOTES.txt

```diff
@@ -1,5 +1,16 @@
+1.23.0
+------
+* New block: Group
+* Add support for upload options in Gallery block
+* Add support for size options in the Image block
+* New block: Button
+* Add scroll support inside block picker and block settings
+* [Android] Fix issue preventing correct placeholder image from displaying during image upload
+* [iOS] Fix diplay of large numbers on ordered lists
+
 1.22.0
 ------
+* Add support for changing images size in Gallery block
```

Comment:
:wave: @mkevins ! Looks like this should be moved up to under `1.23.0`. Does that seem right to you?

Reply:
Actually, that was released as part of 1.22.0, so it's correct there, but I just noticed it's listed twice under 1.22.0 :thinking: I guess one of the duplicates under 1.22.0 can be removed.